news-bundle/src/Controller/Page/NewsFeedController.php

```diff
@@ -75,6 +75,7 @@ public function __invoke(Request $request, PageModel $pageModel): Response
             $this->container->get('event_dispatcher')->dispatch($event, TransformArticleForFeedEvent::NAME);
             $feed->add($event->getItem());
             $this->tagResponse($article);
+            $this->tagResponse('contao.db.tl_news_archive.'.$article->pid);
```

Comment:
Not sure what happens if we tag the response with the same tag multiple times? @Toflar